go/tests/api_test.go

```diff
@@ -45,7 +45,7 @@ type test struct {
 const hostPort = "http://localhost:3100"
 
 func TestReflectionAPI(t *testing.T) {
-	filenames, err := filepath.Glob(filepath.FromSlash("../../tests/*.yaml"))
+	filenames, err := filepath.Glob(filepath.FromSlash("../../tests/reflection_api_tests.yaml"))
```

Comment:
@apascal07 is it okay if I remove the wildcard from here?

Reply:
Yes